ruma-client-api/src/r0/account/change_password.rs

```diff
@@ -18,6 +18,9 @@ ruma_api! {
         /// The new password for the account.
         pub new_password: String,
 
+        /// Whether the user's other access tokens, and their associated devices, should be revoked if the request succeeds.
+        pub logout_devices: Option<bool>,
```

Comment:
Given there's only two cases to distinguish here (since `None` is equivalent to `Some(true)`), this should be a `bool` with the right serde attributes so an absent fields deserializes to `true`, and a value of `true` is not serialized; see for example `ruma-events/src/room/server_acl.rs` where the `allow_ip_literals` has the same semantics.

Reply:
Ok I understand, thank you ill take a look